save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsView.kt

```diff
@@ -153,7 +151,7 @@ abstract class UserSettingsView : AbstractView<UserSettingsProps, UserSettingsVi
             }
             imageUpload = { file ->
                 scope.launch {
-                    postImageUpload(file, props.userName!!, AvatarType.USER, ::noopLoadingHandler)
+                    postImageUpload(file, props.userInfo?.name!!, AvatarType.USER, ::noopLoadingHandler)
```

Comment:
Are you sure you want to use `!!` operator? Sometimes `props.userInfo` might be null as props might not be initialised yet